### DISCUSSION TITLE
How can i import code from repository in discussion?

### DISCUSSION THREAD

**Discussion by osangu** [community/community#124919] [Discussions] [answered]
### Select Topic Area

Question

### Body

I want to import code from repository like in issues.

How can i do?

**Comment by angelo-manalo** [comment, marked as answer]
1. **Locate the Code:**
   - Go to your GitHub repository and find the file with the code you want to reference.

2. **Highlight the Code:**
   - Click on the line number(s) on the left side of the code you want to highlight. This will highlight the code.

3. **Copy the Permalink:**
   - After highlighting the code, a little link icon will appear next to the line numbers. Click this link icon to copy the URL (permalink) to your clipboard.

4. **Create a New Issue:**
   - Go to the "Issues" tab in your repository.
   - Click on "New issue" to create a new issue.

5. **Paste the Link in the Issue:**
   - In the issue description, paste the URL you copied. GitHub will automatically convert this link into a reference to the code snippet.
   - Add any additional details or descriptions about the issue.

Here’s an example of what the issue might look like:

```
### Issue Title

Description of the issue...

#### Code Reference

Here is the problematic code:

https://github.com/username/repo/blob/main/path/to/file#L10-L20
```

**Reply by osangu** [reply]
Thank you!! 

<img width="753" alt="스크린샷 2024-05-21 오후 3 22 07" src="https://github.com/community/community/assets/80697064/d667cabe-a244-48a9-baef-e5053a9907ca">

**Reply by angelo-manalo** [reply]
You're welcome :) @osangu
